Add new CLI commands (#29495)
dotnet/docs: docs/machine-learning/automate-training-with-cli.md

- regression
- recommendation
- image classification

> - train
> - forecasting

Reviewer: MInor nit, but I'd switch the order of these two.

```suggestion
- forecasting
- train
```

The ML.NET CLI can be installed based on your operating system and its architecture with the following command:

```console
dotnet tool install --global mlnet-<OS>-<ARCH>
```

For example, the x64 version of Windows can be installed with:

```console
dotnet tool install --global mlnet-win-x64
```

> More information on installing the ML.NET CLI can be found [here](./how-to-guides/install-ml-net-cli.md).

Reviewer: ```suggestion
For more information, see the [Install the ML.NET CLI](./how-to-guides/install-ml-net-cli.md) how-to guide.
```
